presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcSplit.java

```diff
@@ -42,6 +52,12 @@ public JdbcSplit(
         return additionalPredicate;
     }
 
+    @JsonProperty
+    public TupleDomain<ColumnHandle> getAdditionalConstraint()
```

Comment:
What are the semantics of this additional constraint? Is it best effort? Or do connectors need to guarantee data they produce satisfies this constraint? If so, then there's an implication for backward compatibility for connectors that don't understand this new method currently.